src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py

```diff
@@ -76,6 +76,65 @@ def body(self):
             ])
         self.cmd('webapp delete -g {} -n {}'.format(self.resource_group, webapp_name))
         #test empty service plan should be automatically deleted.
-        result = self.cmd('webapp plan list -g {}'.format(self.resource_group), checks=[
+        result = self.cmd('app-service-plan list -g {}'.format(self.resource_group), checks=[
             JMESPathCheck('length(@)', 0)
             ])
+
+class WebappConfigureTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(WebappConfigureTest, self).__init__(__file__, test_method)
+        self.resource_group = 'azurecli-webapp-config'
+        self.webapp_name = 'webapp-config-test'
```

Comment:
Since the webapp creation is not part of the test, it should be part of setup so that any time it might take will not be incurred when playing back the test. 

---
In reply to: [79496513](https://github.com/Azure/azure-cli/pull/924#discussion_r79496513) [](ancestors = 79496513)